src/chitterApi.js

```diff
@@ -0,0 +1,103 @@
+class chitterApi {
+  constructor() {
+    this.apiUrl = "https://chitter-backend-api-v2.herokuapp.com/peeps";
+    this.apiCreateUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/users";
+    this.apiLoginUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/sessions";
+    this.apiCreatePeppUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps";
+    this.apiLikePeepUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps/peepId/likes/userId";
+    this.apiDeletePeepUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps/peepId";
+  }
+  async fetchAll() {
+    return await fetch(this.apiUrl)
+      .then((response) => response.json())
+      .then((response) => {
+        return response;
+      });
+  }
+
+  async createUser(_user, _password) {
+    return await fetch(this.apiCreateUserUrl, {
+      method: "POST",
+      headers: {
+        "Content-type": "application/json",
+      },
+      body:
+        '{ "user": { "handle": "' +
+        _user +
+        '", "password": "' +
+        _password +
+        '" } }',
+    })
+      .then((response) => response.json())
+      .then((response) => {
+        return response;
+      });
```

Comment:
Why not reuse your fetchall method but with an optional data param?

Reply:
Another old habit is to make functions work too hard by adding constrains so I wanted to stay away from that to the point of refusing to do the right thing